formOKCancel форма OK Отмена

AZANIR opened the pull request:
//окно с вводом текста и кнопкой ОК

AZANIR committed:
formOKCancel форма OK Отмена

//окно с вводом текста и кнопкой ОК

lord-alfred commented on Сниппеты/[Лог, Оповещения]/formOKCancel форма OK Отмена.cs:
```diff
@@ -0,0 +1,20 @@
+﻿//окно с вводом текста и кнопкой ОК 
```
Нужно ниже написать комментарий о том, что необходимо создать zenno-переменную с именем `answer`, куда попадет результат из поля ввода

lord-alfred replied:
Нужно переимновать сниппет в "Окно ввода строки.cs" и положить в новую коллекцию: [Сторонний GUI]

lord-alfred reviewed (changes requested):
Необходимо добавить описание про переменную, куда попадет результат

lord-alfred commented:
@Zymlex , расположение сниппета корректное? Меня название смущает, но я не знаю как лучше по-русски написать его. Посоветуй?

**UDP**: там только кнопка OK, там нет кнопки Cancel/Отмена.

ghost commented:
> @Zymlex , расположение сниппета корректное? Меня название смущает, но я не знаю как лучше по-русски написать его. Посоветуй?

Либо [GUI], либо [Графический интерфейс], более правильней [Сторонний графический интерфейс], но длинно, а первые два варианта довольно обширны.

ghost commented:
> Посоветуй?

Напиши, [Сторонний GUI].

lord-alfred commented:
> Либо [GUI], либо [Графический интерфейс], более правильней [Сторонний графический интерфейс], но длинно, а первые два варианта довольно обширны.

А по поводу названия сниппета что думаешь? `formOKCancel форма OK Отмена` мне кажется, что плохое название, тем более там в форме нет Cancel

AZANIR committed:
DOM pageReload перезагрузить страницу

//мягкая перезагрузка, F5
instance.ActiveTab.MainDocument.EvaluateScript("window.location.reload(false)");

//жесткая перезагрузка, как Ctrl+F5
instance.ActiveTab.MainDocument.EvaluateScript("window.location.reload(true)");

AZANIR committed:
Update DOM pageReload перезагрузить страницу.cs

ghost commented:
"Окно запроса строки.cs", но там ещё не хватает упоминания про GAC с Forms. Я бы ещё отформатировал красивее, названия переменным подправил бы и оформил под общий код.

lord-alfred commented:
GAC там не нужен, я проверял. 
Форматирование можно улучшить, но ладно - пока четких стандартов для репозитория нету, а в дальнейшем посмотрим как автоформатирование прикрутить (например). 
По поводу общего кода - тут это лишнее, мне кажется. Объяснять дольше сразу же что и куда положить, а так проще сниппет заюзать будет.

Может название лучше `Окно ввода строки.cs`?

ghost commented:
> Может название лучше `Окно ввода строки.cs`?

Можно и так.

lord-alfred commented on Сниппеты/[Браузер]/DOM pageReload перезагрузить страницу.cs:
```diff
@@ -0,0 +1,5 @@
+//мягкая перезагрузка, F5
```
Нужно переименовать сниппет в: "Перезагрузить страницу.cs"

AZANIR committed:
Update and rename formOKCancel форма OK Отмена.cs to formOK форма OK.cs

AZANIR committed:
Rename Сниппеты/[Браузер]/DOM pageReload перезагрузить страницу.cs to Сниппеты/[Браузер]Перезагрузить страницу.cs

AZANIR committed:
Update and rename formOK форма OK.cs to Окно ввода строки.cs

текстовые правки и название

AZANIR committed:
Update Окно ввода строки.cs

AZANIR committed:
Rename Сниппеты/[Браузер]Перезагрузить страницу.cs to Сниппеты/[Браузер]/Перезагрузить страницу.cs

AZANIR committed:
Rename Сниппеты/[Лог, Оповещения]/Окно ввода строки.cs to Сниппеты/Сниппеты/[Сторонний GUI]/Окно ввода строки.cs

AZANIR committed:
Rename Сниппеты/Сниппеты/[Сторонний GUI]/Окно ввода строки.cs to Сниппеты/[Сторонний GUI]/Окно ввода строки.cs
